README.md

```diff
@@ -72,7 +72,7 @@ Packages are available for
 * Debian: See the [whonix/monero-gui repository](https://gitlab.com/whonix/monero-gui#how-to-install-monero-using-apt-get)
 * Void Linux: `xbps-install -S monero-gui`
 * GuixSD: `guix package -i monero-gui`
-* macOS (homebrew): `brew cask install monero-wallet`
+* macOS (homebrew): `brew install --cask monero-wallet`
```

Comment:
Tested on my mac machine, the brew install --cask monero-wallet worked for me, but brew cask install monero-wallet did not.

Reply:
Looks like they updated the syntax, thanks for correcting it.